app/controllers/sessions_controller.rb

```diff
@@ -14,7 +14,12 @@ def create
   end
 
   def destroy
-    log_out
+    log_out if logged_in?
     redirect_to root_path
   end
+
+  def handling user, par
```

Comment:
Thường thì, việc tách method là để cho code được tổ chức tốt hơn, clear về cấu trúc, hoặc tránh lặp code
Tuy nhiên:
  - em đang đặt tên method và danh sách tham số khá ngắn gọn, ko dễ hiểu
  - đoạn xử lý logic này hình như chỉ xuất hiện 1 lần duy nhất trong mã nguồn, em tách thành method như này là có cần thiết ko?

Reply:
Rubocop báo lỗi "Assignment Branch Condition size for create is too high" em mới tách ra ạ!